### PR TITLE
fix: cleanup and fix samples

### PR DESCRIPTION
**Pull request by JustinBeckwith**
When trying to nail down #891, I noticed that most of the samples were in disrepair.  To be able to effectively test the new auth lib, we should probably land this first.

### CODE REVIEW

File: samples/blogger/blogger.js [hunk 1]
```diff
@@ -13,18 +13,16 @@
 
 'use strict';
 
-var google = require('../../lib/googleapis');
-var blogger = google.blogger('v3');
-var util = require('util');
+const google = require('googleapis');
+const blogger = google.blogger('v3');
+const config = require('../config.json');
```
Comment (alexander-fenster) [inline samples/blogger/blogger.js]: Normally we read api key from the environment variable, which makes it easier to test the samples code in the continuous integration (which we don't do now anyway). If it's too much change, I'm ok to leave it in the file.

File: samples/defaultauth.js [hunk 2]
```diff
@@ -24,19 +24,20 @@ var compute = google.compute('v1');
  * method rather than creating your own JWT or Compute client directly.
  *
  * Note: In order to run on a local developer machine, it is necessary to download a private key
- * file to your machine, and to set a local environment variable pointing to the location of the
+ * file to your machine, and to set a local environment constiable pointing to the location of the
```
Comment (alexander-fenster) [inline samples/defaultauth.js]: I'm not a native English speaker but it looks like replacing all `var` with `const` caused some unexpected consequences!

Comment (JustinBeckwith) [reply samples/defaultauth.js]: *laughs* oops.  Nice catch.

File: samples/blogger/blogger.js (hunk 1 above)
Comment (JustinBeckwith) [reply samples/blogger/blogger.js]: It's not a huge deal.  Typically if I'm going to do that, I like to use `nconf` so I can support both file based and env based methods.  I'll take a shot at that tonight.

**Comment by alexander-fenster** [comment]
Unrelated question: do we need to protect the master branch for this repo to require at least the Travis CI to pass (not mentioning circle)?

**Comment by JustinBeckwith** [comment]
YES.  We should absolutely enable master branch protection.

**Comment by alexander-fenster** [comment]
I have enabled branch protection for branch master in this repo.

**Pull request merged**